cfr.yaml

```diff
@@ -719,3 +719,60 @@
     notification_text:
       string_id: cfr-doorhanger-feature-notification
     category: cfrFeatures
+- id: HOMEPAGE_REMEDIATION
+  groups:
+    - cfr
+  frequency:
+    lifetime: 3
+  # Has google set as the homepage, has CFR features enabled, has profile
+  # older than 6 months
+  targeting: "!homePageSettings.isDefault && homePageSettings.isCustomUrl && homePageSettings.urls[.host == 'google.com']|length > 0 && visitsCount >= 3 && userPrefs.cfrFeatures && currentDate|date - profileAgeCreated > 15770000000 && firefoxVersion > 82 && localeLanguageCode == 'en'"
```

Comment:
Just to confirm that are we looking to show this CFR for all regions?

Reply:
Good question. I'll make it US only at first and we can discuss this afterwards.